### PR TITLE
288487 - Create the landing page

### PR DESCRIPTION
**Pull request by FujaleAhmedShaik**
### What one thing this PR does?

This will create the NCEA landing page with hero content and a video block.

### Task details
- [288487 - Create the landing page](https://dev.azure.com/defragovuk/DEFRA-NCEA/_workitems/edit/288487)

### Other notes

- Now we have placed some related video from the Defra UK YouTube account. The actual video is not yet ready.
- The search block on the landing page will be developed as part of another [ticket](https://dev.azure.com/defragovuk/DEFRA-NCEA/_workitems/edit/292601).
- The figma design that is related can be found [here](https://www.figma.com/file/OmE5IlvwbGrYDMBM8P1U4M/%F0%9F%92%9A-NCEA-ALPHA-MVP-Prototype?node-id=3371%3A16279&mode=dev)
- We have a [task](https://dev.azure.com/defragovuk/DEFRA-NCEA/_workitems/edit/288418) to customize the header and footer.

### How do these changes look like?

**Large screens**
| Large screens |
|-|

**Tablet screen**
| Tablet screen |
|-|

**Mobile screen**
| Mobile screen |
|-|

### Dev-tested in

1. Local environment  

- [Home/Landing page](http://localhost:3000/)

2. Dev environment 

- Page name/link 1. 
- ...

### CODE REVIEW

File: src/assets/sass/application.scss [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+$gov-defra-color-websafe: govuk-organisation-colour(
+  'department-for-environment-food-rural-affairs'
+);
+
+.app-main-class {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
```
Comment (karthigayayiniram) [inline src/assets/sass/application.scss:11]: please keep all scss files in assets/sass folder itself. index.scss can import other partial scss files. rename this file as partial.

File: tests/routes/web/home.spec.ts [hunk 2]
```diff
@@ -21,10 +26,19 @@ describe('Home Routes', () => {
...
-      url: '/',
+      url: webRoutePaths.home,
     };
 
     const response = await server.inject(options);
+    const rawHTML = response.payload;
+    const parser = new DOMParser();
+    const document = parser.parseFromString(rawHTML, 'text/html');
     expect(response.statusCode).toEqual(200);
+    expect(
```
Comment (karthigayayiniram) [inline tests/routes/web/home.spec.ts:37]: Can you group the assertions in a test case and group the test cases in a test suite.

File: tests/routes/web/home.spec.ts [hunk 3]
```diff
@@ -21,10 +26,19 @@ describe('Home Routes', () => {
...
+      document?.querySelector('.banner-container__heading-xl')?.textContent
+    ).toBe('Natural Capital Search Service');
+    expect(
+      document?.querySelector('.video-container__caption-m')?.textContent
+    ).toBe('What is natural capital?');
```
Comment (karthigayayiniram) [inline tests/routes/web/home.spec.ts:42]: Please add another test case for comparing landing page snapshot.

Comment (FujaleAhmedShaik) [reply tests/routes/web/home.spec.ts]: Now snapshot test case has been added

File: tests/routes/web/home.spec.ts (hunk 2 above)
Comment (FujaleAhmedShaik) [reply tests/routes/web/home.spec.ts]: Yes, now we have grouped the test cases.

File: src/assets/sass/application.scss (hunk 1 above)
Comment (FujaleAhmedShaik) [reply src/assets/sass/application.scss]: Incorporated as per the discussion

**Review by ganesh-rajamahanthy — APPROVED** [review]

**Review by karthigayayiniram — APPROVED** [review]

**Pull request merged**